x-pack/plugins/grokdebugger/public/components/grok_debugger/grok_debugger.js

```diff
@@ -126,33 +118,31 @@ export class GrokDebuggerComponent extends React.Component {
     return (
       <EuiPage>
         <EuiPageBody>
-          <EuiPageContent>
-            <EuiPageContentBody>
-              <EuiForm className="grokdebugger-container" data-test-subj="grokDebuggerContainer">
-                <EventInput value={this.state.rawEvent} onChange={this.onRawEventChange} />
-                <PatternInput value={this.state.pattern} onChange={this.onPatternChange} />
-                <EuiSpacer />
-                <CustomPatternsInput
-                  value={this.state.customPatterns}
-                  onChange={this.onCustomPatternsChange}
+          <EuiPageSection>
```

Comment:
Looks like there's maybe a regression here with the page background

Before:
<img width="1811" alt="Screenshot 2023-10-12 at 12 35 59 PM" src="https://github.com/elastic/kibana/assets/5226211/aaa27670-2f56-4b55-bb55-305701ef0cf0">

After:
<img width="1814" alt="Screenshot 2023-10-12 at 12 36 19 PM" src="https://github.com/elastic/kibana/assets/5226211/337f40a3-8882-43a8-a7e7-129845680ca0">

Reply:
Should be fixed by https://github.com/elastic/kibana/pull/168300/commits/68c1167d0dcaa67999f4683610ebd4c58244f8ed